### PR TITLE
Bug 1132472 - Task detail page should display after creating a task

### PR DESCRIPTION
**Pull request by tessie**

### CODE REVIEW

**Comment by tessie** [comment]
@bobsilverberg: Ready for review

File: oneanddone/tasks/views.py [hunk 1]
```diff
@@ -149,10 +149,10 @@ def get_context_data(self, *args, **kwargs):
         return ctx
 
     def form_valid(self, form):
-        form.save(self.request.user)
-
+        task = form.save(self.request.user)
+
```
Comment (bobsilverberg) [inline oneanddone/tasks/views.py]: This blank line contains whitespace. Your `pep8` or `flake8` checker should have caught that. Do you have a pre-commit hook set up to check for `pep8` errors?

**Comment by bobsilverberg** [comment]
This works well. Thanks @tessie. The only issue is that `pep8` issue with the whitespace on a blank line. Please fix that and then squash the commits and this should be ready to merge.

**Comment by tessie** [comment]
@bobsilverberg .Updated

**Comment by tessie** [comment]
Thank you for the review

**Comment by bobsilverberg** [comment]
Thanks @tessie. Looks good.

**Pull request merged**